src/components/dashboard/SendLinkSummary.js

```diff
@@ -187,19 +187,19 @@ const SendLinkSummary = ({ screenProps, styles }: AmountProps) => {
             </View>
           </Section.Row>
           <Section.Title fontWeight="medium">YOU ARE SENDING</Section.Title>
-          <Section.Title fontWeight="medium" style={styles.amountWrapper}>
+          <Section.Row fontWeight="medium" style={styles.amountWrapper}>
             <BigGoodDollar
               number={amount}
               color="red"
               bigNumberProps={{
                 fontSize: 36,
-                lineHeight: 24,
+                lineHeight: 36,
```

Comment:
Sure this isn't affecting web?

Reply:
The lineHeight should be equal or bigger than FontSize, on the contrary the letters are cut. Anyway I'll look that on web.